hack/verify-starlark.sh

```diff
@@ -17,6 +17,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+SCRIPT_DIR="$(dirname ${BASH_SOURCE[0]})"
```

Comment:
`${BASH_SOURCE[0]}` needs double quote escaping here to appease the linter

```suggestion
SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
```

Reply:
Fixed!